README.md

```diff
@@ -227,4 +228,23 @@ Copy the UUID and register the device in the [Ubirch Console Web Interface](http
 - evaluate the message response, see [ubirch_parse_response()](https://github.com/ubirch/ubirch-esp32-api-http/blob/master/response.h#L42)
 - react to the UI message response parameter "i" to turn on the blue LED, if the value is above 1000, see [app_main()](https://github.com/ubirch/example-esp32/blob/master/main/main.c#L67-L69) 
 
+## Build and run tests
 
+To build and run the test runner, that controls tests that are implemented in the components, you need to have the esp-idf as well as the xtensa toolchain installed as described above.
+Similar to the [build](#build) section run the following commands to prepare the build (note that you need to change to the `test` directory):
```

Comment:
Maybe it is worth noting, which `test` directory? You have to go to the `test` directory from the project root, is that right? Otherwise one might think to go to the test directories from the components.

Reply:
Yes, you are right! I added a note.